.github/workflows/library-redis7.0.yaml

```diff
@@ -85,3 +85,65 @@ jobs:
       with:
         name: index.unikraft.io/unikraft.org/redis:7.0
         push: true
+
+  run-remote:
+    name: Test redis:7.0 (Remote OCI)
+    needs: [build, push]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to OCI registry
+        uses: docker/login-action@v3
+        with:
+          registry: index.unikraft.io
+          username: ${{ secrets.REG_USERNAME }}
+          password: ${{ secrets.REG_TOKEN }}
+
+      - name: Pull, run, validate and cleanup unikernel
+        run: |
+          set -euo pipefail
+          echo "Pull and start redis:7.0 unikernel"
+          IMAGE=index.unikraft.io/unikraft.org/redis:7.0
+          docker pull "$IMAGE"
+          CONTAINER_ID=$(docker run --rm -d "$IMAGE")
+          echo "Wait and validate container startup"
+          sleep 5
+          if ! docker ps --filter "id=$CONTAINER_ID" --format '{{.ID}}' | grep -q .; then
+            echo "ERROR: redis:7.0 unikernel is not running" >&2
+            docker logs "$CONTAINER_ID" >&2 || true
+            docker stop "$CONTAINER_ID" || true
+            exit 1
+          fi
+          echo "redis:7.0 unikernel started successfully"
+          echo "Cleanup container"
+          docker stop "$CONTAINER_ID" || true
+
+  run-local:
+    name: Test redis:7.0 (Local build)
+    needs: [build]
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install kraft CLI
+        run: |
+          echo "deb [trusted=yes] https://deb.pkg.kraftkit.sh/ /" | sudo tee /etc/apt/sources.list.d/kraftkit.list
+          sudo apt-get update
+          sudo apt-get install -y kraftkit
```

Comment:
You can use the kraftkit github action so there is no need to install, like we discussed

<https://github.com/marketplace/actions/build-unikernel-images-with-unikraft#use-in-github-actions>